.github/workflows/pr-validation.yml

```diff
@@ -0,0 +1,54 @@
+name: Pull Request Validation
+
+on:
+  pull_request:
+    branches:
+      - trunk
+    types:
+      - "labeled"
+      - "unlabeled"
+      - "opened"
+      - "reopened"
+      - "synchronize"
```

Comment:
Too bad indeed that there's no `milestoned` (or similar) activity type in https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull_request. I guess for now one would have to add/remove/add labels to re-trigger "manually".